libraries/botbuilder-middleware-aws-comprehend/src/schema.ts

```diff
@@ -0,0 +1,9 @@
+/**
+ * @module botbuildercommunity/middleware-aws-comprehend
+ */
+
+export interface AWSComprehendOptions {
+    apiVersion: string;
```

Comment:
Why not make this one optional? I think it is hard for endusers to choose the right API version and API version changes could also have breaking changes, which will break this middleware.

Reply:
Inherited from `Comprehend.ClientConfiguration`.